### ISSUE TITLE
Crash after ending any playback.

### ISSUE THREAD

**Issue by rohitghali** [jellyfin/jellyfin-androidtv#3146]
### Describe the bug

After playing a video and closing it, jellyfin-androidtv just crashes.

### Logs

---
client: Jellyfin for Android TV
client_version: 0.15.12
client_repository: https://github.com/jellyfin/jellyfin-androidtv
type: crash_report
format: markdown
---
### Logs

***Stack Trace***: 
```log
java.lang.NullPointerException: Attempt to invoke virtual method 'java.util.UUID org.jellyfin.sdk.model.api.UserDto.getId()' on a null object reference
	at org.jellyfin.androidtv.ui.playback.ExternalPlayerActivity.markPlayed(ExternalPlayerActivity.java:268)
	at org.jellyfin.androidtv.ui.playback.ExternalPlayerActivity$2.onClick(ExternalPlayerActivity.java:185)
	at com.android.internal.app.AlertController$ButtonHandler.handleMessage(AlertController.java:174)
	at android.os.Handler.dispatchMessage(Handler.java:106)
	at android.os.Looper.loop(Looper.java:223)
	at android.app.ActivityThread.main(ActivityThread.java:7719)
	at java.lang.reflect.Method.invoke(Native Method)
	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:592)
	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:947)

```  
***Logcat***: 
```log
10-26 16:47:41.375 E/AndroidRuntime(30702): 	at org.jellyfin.androidtv.ui.itemdetail.FullDetailsFragment$3.run(FullDetailsFragment.java:272)
10-26 16:47:41.375 E/AndroidRuntime(30702): 	at android.os.Handler.handleCallback(Handler.java:938)
10-26 16:47:41.375 E/AndroidRuntime(30702): 	at android.os.Handler.dispatchMessage(Handler.java:99)
10-26 16:47:41.375 E/AndroidRuntime(30702): 	at android.os.Looper.loop(Looper.java:223)
10-26 16:47:41.375 E/AndroidRuntime(30702): 	at android.app.ActivityThread.main(ActivityThread.java:7719)
10-26 16:47:41.375 E/AndroidRuntime(30702): 	at java.lang.reflect.Method.invoke(Native Method)
10-26 16:47:41.375 E/AndroidRuntime(30702): 	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:592)
10-26 16:47:41.375 E/AndroidRuntime(30702): 	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:947)
10-26 17:18:29.818 E/AndroidRuntime( 1177): FATAL EXCEPTION: main
10-26 17:18:29.818 E/AndroidRuntime( 1177): Process: org.jellyfin.androidtv, PID: 1177
10-26 17:18:29.818 E/AndroidRuntime( 1177): java.lang.NullPointerException: Attempt to invoke virtual method 'java.util.UUID org.jellyfin.sdk.model.api.UserDto.getId()' on a null object reference
10-26 17:18:29.818 E/AndroidRuntime( 1177): 	at org.jellyfin.androidtv.ui.itemdetail.FullDetailsFragment$3.run(FullDetailsFragment.java:272)
10-26 17:18:29.818 E/AndroidRuntime( 1177): 	at android.os.Handler.handleCallback(Handler.java:938)
10-26 17:18:29.818 E/AndroidRuntime( 1177): 	at android.os.Handler.dispatchMessage(Handler.java:99)
10-26 17:18:29.818 E/AndroidRuntime( 1177): 	at android.os.Looper.loop(Looper.java:223)
10-26 17:18:29.818 E/AndroidRuntime( 1177): 	at android.app.ActivityThread.main(ActivityThread.java:7719)
10-26 17:18:29.818 E/AndroidRuntime( 1177): 	at java.lang.reflect.Method.invoke(Native Method)
10-26 17:18:29.818 E/AndroidRuntime( 1177): 	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:592)
10-26 17:18:29.818 E/AndroidRuntime( 1177): 	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:947)
10-27 13:26:14.515 E/AndroidRuntime( 6282): FATAL EXCEPTION: main
10-27 13:26:14.515 E/AndroidRuntime( 6282): Process: org.jellyfin.androidtv, PID: 6282
10-27 13:26:14.515 E/AndroidRuntime( 6282): java.lang.NullPointerException: Attempt to invoke virtual method 'java.util.UUID org.jellyfin.sdk.model.api.UserDto.getId()' on a null object reference
10-27 13:26:14.515 E/AndroidRuntime( 6282): 	at org.jellyfin.androidtv.ui.itemdetail.FullDetailsFragment$3.run(FullDetailsFragment.java:272)
10-27 13:26:14.515 E/AndroidRuntime( 6282): 	at android.os.Handler.handleCallback(Handler.java:938)
10-27 13:26:14.515 E/AndroidRuntime( 6282): 	at android.os.Handler.dispatchMessage(Handler.java:99)
10-27 13:26:14.515 E/AndroidRuntime( 6282): 	at android.os.Looper.loop(Looper.java:223)
10-27 13:26:14.515 E/AndroidRuntime( 6282): 	at android.app.ActivityThread.main(ActivityThread.java:7719)
10-27 13:26:14.515 E/AndroidRuntime( 6282): 	at java.lang.reflect.Method.invoke(Native Method)
10-27 13:26:14.515 E/AndroidRuntime( 6282): 	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:592)
10-27 13:26:14.515 E/AndroidRuntime( 6282): 	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:947)
--------- beginning of system
10-27 23:17:28.361 W/ViewRootImpl[StartupActivity](22435): Cancelling event due to no window focus: KeyEvent { action=ACTION_UP, keyCode=KEYCODE_BACK, scanCode=158, metaState=0, flags=0x28, repeatCount=0, eventTime=119571741, downTime=119571575, deviceId=12, source=0x301, displayId=-1 }
10-27 23:17:28.417 E/AndroidRuntime(22435): FATAL EXCEPTION: main
10-27 23:17:28.417 E/AndroidRuntime(22435): Process: org.jellyfin.androidtv, PID: 22435
10-27 23:17:28.417 E/AndroidRuntime(22435): java.lang.NullPointerException: Attempt to invoke virtual method 'java.util.UUID org.jellyfin.sdk.model.api.UserDto.getId()' on a null object reference
10-27 23:17:28.417 E/AndroidRuntime(22435): 	at org.jellyfin.androidtv.ui.itemdetail.FullDetailsFragment$3.run(FullDetailsFragment.java:272)
10-27 23:17:28.417 E/AndroidRuntime(22435): 	at android.os.Handler.handleCallback(Handler.java:938)
10-27 23:17:28.417 E/AndroidRuntime(22435): 	at android.os.Handler.dispatchMessage(Handler.java:99)
10-27 23:17:28.417 E/AndroidRuntime(22435): 	at android.os.Looper.loop(Looper.java:223)
10-27 23:17:28.417 E/AndroidRuntime(22435): 	at android.app.ActivityThread.main(ActivityThread.java:7719)
10-27 23:17:28.417 E/AndroidRuntime(22435): 	at java.lang.reflect.Method.invoke(Native Method)
10-27 23:17:28.417 E/AndroidRuntime(22435): 	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:592)
10-27 23:17:28.417 E/AndroidRuntime(22435): 	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:947)
10-27 23:17:28.362 I/chatty  (22435): uid=10228(org.jellyfin.androidtv) identical 7 lines
10-27 23:17:28.362 W/ViewRootImpl[StartupActivity](22435): Cancelling event due to no window focus: KeyEvent { action=ACTION_UP, keyCode=KEYCODE_BACK, scanCode=158, metaState=0, flags=0x28, repeatCount=0, eventTime=119571741, downTime=119571575, deviceId=12, source=0x301, displayId=-1 }
10-27 23:17:32.987 W/ViewRootImpl[StartupActivity](29509): Dropping event due to no window focus: KeyEvent { action=ACTION_DOWN, keyCode=KEYCODE_DPAD_CENTER, scanCode=96, metaState=0, flags=0x8, repeatCount=0, eventTime=119574855, downTime=119574855, deviceId=12, source=0x301, displayId=-1 }
10-27 23:17:33.005 W/ViewRootImpl[StartupActivity](29509): Cancelling event due to no window focus: KeyEvent { action=ACTION_UP, keyCode=KEYCODE_DPAD_CENTER, scanCode=96, metaState=0, flags=0x28, repeatCount=0, eventTime=119574998, downTime=119574855, deviceId=12, source=0x301, displayId=-1 }
10-27 23:17:33.015 I/chatty  (29509): uid=10228(org.jellyfin.androidtv) identical 7 lines
10-27 23:17:33.015 W/ViewRootImpl[StartupActivity](29509): Cancelling event due to no window focus: KeyEvent { action=ACTION_UP, keyCode=KEYCODE_DPAD_CENTER, scanCode=96, metaState=0, flags=0x28, repeatCount=0, eventTime=119574998, downTime=119574855, deviceId=12, source=0x301, displayId=-1 }
10-27 23:36:30.523 W/ViewRootImpl[StartupActivity](29509): Cancelling event due to no window focus: KeyEvent { action=ACTION_UP, keyCode=KEYCODE_BACK, scanCode=158, metaState=0, flags=0x28, repeatCount=0, eventTime=120713774, downTime=120713593, deviceId=12, source=0x301, displayId=-1 }
10-27 23:36:30.524 I/chatty  (29509): uid=10228(org.jellyfin.androidtv) identical 7 lines
10-27 23:36:30.524 W/ViewRootImpl[StartupActivity](29509): Cancelling event due to no window focus: KeyEvent { action=ACTION_UP, keyCode=KEYCODE_BACK, scanCode=158, metaState=0, flags=0x28, repeatCount=0, eventTime=120713774, downTime=120713593, deviceId=12, source=0x301, displayId=-1 }
10-27 23:36:30.658 E/AndroidRuntime(29509): FATAL EXCEPTION: main
10-27 23:36:30.658 E/AndroidRuntime(29509): Process: org.jellyfin.androidtv, PID: 29509
10-27 23:36:30.658 E/AndroidRuntime(29509): java.lang.NullPointerException: Attempt to invoke virtual method 'java.util.UUID org.jellyfin.sdk.model.api.UserDto.getId()' on a null object reference
10-27 23:36:30.658 E/AndroidRuntime(29509): 	at org.jellyfin.androidtv.ui.itemdetail.FullDetailsFragment$3.run(FullDetailsFragment.java:272)
10-27 23:36:30.658 E/AndroidRuntime(29509): 	at android.os.Handler.handleCallback(Handler.java:938)
10-27 23:36:30.658 E/AndroidRuntime(29509): 	at android.os.Handler.dispatchMessage(Handler.java:99)
10-27 23:36:30.658 E/AndroidRuntime(29509): 	at android.os.Looper.loop(Looper.java:223)
10-27 23:36:30.658 E/AndroidRuntime(29509): 	at android.app.ActivityThread.main(ActivityThread.java:7719)
10-27 23:36:30.658 E/AndroidRuntime(29509): 	at java.lang.reflect.Method.invoke(Native Method)
10-27 23:36:30.658 E/AndroidRuntime(29509): 	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:592)
10-27 23:36:30.658 E/AndroidRuntime(29509): 	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:947)
--------- beginning of main
10-28 01:06:41.043 I/org.jellyfin.sdk.api.sockets.OkHttpWebsocketSession(21048): Sending (raw) message {"MessageType":"KeepAlive","Data":{}}
10-28 01:06:41.061 I/org.jellyfin.sdk.api.sockets.OkHttpWebsocketSession(21048): Receiving (raw) message {"MessageType":"KeepAlive","MessageId":"005f2772c466446f906439dc5a860af4"}
10-28 01:07:09.385 I/jellyfin-apiclient(21048): ReportPlaybackStopped: Item d79b1056-c738-e91f-1c96-54ced7f0f3ac, Ticks: 0
10-28 01:07:09.629 I/ProcessLifecycleInitializer$create(21048): Process lifecycle started
10-28 01:07:09.640 I/SessionRepositoryImpl(21048): Restoring session
10-28 01:07:09.644 I/SessionRepositoryImpl(21048): Destroying current session
10-28 01:07:09.658 I/jellyfin-apiclient(21048): Response received from: http://rohitghali.ddns.net:8096/Sessions/Playing/Stopped
10-28 01:07:09.718 E/libEGL  (21048): eglSurfaceAttribImpl:1562 error 300d (EGL_BAD_SURFACE)
10-28 01:07:10.085 I/org.jellyfin.sdk.api.sockets.OkHttpWebsocketSession(21048): Receiving (raw) message {"MessageType":"UserDataChanged","MessageId":"cbad8ba8ddf24f28aafeacacfb5b1bab","Data":{"UserId":"3d8ce2f1aed34791b0a12d37ec8ab191","UserDataList":[{"PlaybackPositionTicks":0,"PlayCount":4,"IsFavorite":false,"LastPlayedDate":"2023-10-28T04:22:45.1466988Z","Played":false,"Key":"4916","ItemId":"d79b1056c738e91f1c9654ced7f0f3ac"},{"UnplayedItemCount":10,"PlaybackPositionTicks":0,"PlayCount":0,"IsFavorite":false,"Played":false,"Key":"c87c13a8-ecbf-9d91-4900-4bc1605cbf69","ItemId":"c87c13a8ecbf9d9149004bc1605cbf69"}]}}
10-28 01:07:10.527 W/Choreographer(21048): Frame time is 0.006724 ms in the future!  Check that graphics HAL is generating vsync timestamps using the correct timebase.
10-28 01:07:11.120 I/org.jellyfin.sdk.api.sockets.OkHttpWebsocketSession(21048): Sending (raw) message {"MessageType":"KeepAlive","Data":{}}
10-28 01:07:11.128 I/org.jellyfin.sdk.api.sockets.OkHttpWebsocketSession(21048): Receiving (raw) message {"MessageType":"KeepAlive","MessageId":"19c2a769095d4887ae758b4540388634"}
10-28 01:07:13.524 E/AndroidRuntime(21048): FATAL EXCEPTION: main
10-28 01:07:13.524 E/AndroidRuntime(21048): Process: org.jellyfin.androidtv, PID: 21048
10-28 01:07:13.524 E/AndroidRuntime(21048): java.lang.NullPointerException: Attempt to invoke virtual method 'java.util.UUID org.jellyfin.sdk.model.api.UserDto.getId()' on a null object reference
10-28 01:07:13.524 E/AndroidRuntime(21048): 	at org.jellyfin.androidtv.ui.playback.ExternalPlayerActivity.markPlayed(ExternalPlayerActivity.java:268)
10-28 01:07:13.524 E/AndroidRuntime(21048): 	at org.jellyfin.androidtv.ui.playback.ExternalPlayerActivity$2.onClick(ExternalPlayerActivity.java:185)
10-28 01:07:13.524 E/AndroidRuntime(21048): 	at com.android.internal.app.AlertController$ButtonHandler.handleMessage(AlertController.java:174)
10-28 01:07:13.524 E/AndroidRuntime(21048): 	at android.os.Handler.dispatchMessage(Handler.java:106)
10-28 01:07:13.524 E/AndroidRuntime(21048): 	at android.os.Looper.loop(Looper.java:223)
10-28 01:07:13.524 E/AndroidRuntime(21048): 	at android.app.ActivityThread.main(ActivityThread.java:7719)
10-28 01:07:13.524 E/AndroidRuntime(21048): 	at java.lang.reflect.Method.invoke(Native Method)
10-28 01:07:13.524 E/AndroidRuntime(21048): 	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:592)
10-28 01:07:13.524 E/AndroidRuntime(21048): 	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:947)
10-28 01:07:13.527 E/ACRA    (21048): ACRA caught a NullPointerException for org.jellyfin.androidtv
10-28 01:07:13.527 E/ACRA    (21048): java.lang.NullPointerException: Attempt to invoke virtual method 'java.util.UUID org.jellyfin.sdk.model.api.UserDto.getId()' on a null object reference
10-28 01:07:13.527 E/ACRA    (21048): 	at org.jellyfin.androidtv.ui.playback.ExternalPlayerActivity.markPlayed(ExternalPlayerActivity.java:268)
10-28 01:07:13.527 E/ACRA    (21048): 	at org.jellyfin.androidtv.ui.playback.ExternalPlayerActivity$2.onClick(ExternalPlayerActivity.java:185)
10-28 01:07:13.527 E/ACRA    (21048): 	at com.android.internal.app.AlertController$ButtonHandler.handleMessage(AlertController.java:174)
10-28 01:07:13.527 E/ACRA    (21048): 	at android.os.Handler.dispatchMessage(Handler.java:106)
10-28 01:07:13.527 E/ACRA    (21048): 	at android.os.Looper.loop(Looper.java:223)
10-28 01:07:13.527 E/ACRA    (21048): 	at android.app.ActivityThread.main(ActivityThread.java:7719)
10-28 01:07:13.527 E/ACRA    (21048): 	at java.lang.reflect.Method.invoke(Native Method)
10-28 01:07:13.527 E/ACRA    (21048): 	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:592)
10-28 01:07:13.527 E/ACRA    (21048): 	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:947)

```  

### App information

***App version***: `0.15.12` (`151299`)  
***Package name***: `org.jellyfin.androidtv`  
***Build***: 
```json
{"BOARD":"karat","BOOTLOADER":"unknown","BRAND":"Amazon","CPU_ABI":"armeabi-v7a","CPU_ABI2":"armeabi","DEVICE":"karat","DISPLAY":"RS8101.1854N","FINGERPRINT":"Amazon\/karat\/karat:11\/RS8101.1854N\/0021810855556:user\/amz-p,release-keys","HARDWARE":"mt8696","HOST":"c5-ri-1804-use1a-b-18-04-fos8-14","ID":"RS8101.1854N","IS_DEBUGGABLE":false,"IS_EMULATOR":false,"MANUFACTURER":"Amazon","MODEL":"AFTKM","PERMISSIONS_REVIEW_REQUIRED":true,"PRODUCT":"karat","RADIO":"unknown","SUPPORTED_32_BIT_ABIS":["armeabi-v7a","armeabi"],"SUPPORTED_64_BIT_ABIS":[],"SUPPORTED_ABIS":["armeabi-v7a","armeabi"],"TAGS":"amz-p,release-keys","TIME":1697733227000,"TYPE":"user","UNKNOWN":"unknown","USER":"build","VERSION":{"ACTIVE_CODENAMES":[],"BASE_OS":"RP1A.200720.009","CODENAME":"REL","INCREMENTAL":"0021810855556","PREVIEW_SDK_FINGERPRINT":"REL","PREVIEW_SDK_INT":0,"RELEASE":"11","RELEASE_OR_CODENAME":"11","SDK":"30","SDK_INT":30,"SECURITY_PATCH":"2020-09-05"}}
```  
***Build config***: 
```json
{"APPLICATION_ID":"org.jellyfin.androidtv","BUILD_TYPE":"release","DEBUG":false,"DEVELOPMENT":false,"VERSION_CODE":151299,"VERSION_NAME":"0.15.12"}
```  

### Device information

***Android version***: `11`  
***Device brand***: `Amazon`  
***Device product***: `karat`  
***Device model***: `AFTKM`  

### Crash information

***Start time***: `2023-10-28T00:22:35.212-04:00`  
***Crash time***: `2023-10-28T01:07:13.606-04:00`  

### Application version

0.15.12

### Where did you install the app from?

Amazon Appstore

### Device information

Latest firestick 4k

### Android version

Android 11, Fire os 

### Jellyfin server version

10.8.11

**Comment by nielsvanvelzen** [comment]
This is likely fixed in the upcoming version (0.16).

**Comment by xero08** [comment]
Encountering the same issue.
